<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>1089</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="is_live_btn">
       <property name="text">
        <string>Live Data</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/record.png</normaloff>icons/record.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plot_btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/play.png</normaloff>icons/play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/delete.png</normaloff>icons/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Load from </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="directory_label">
       <property name="text">
        <string>pico_app/data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_dir_btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/folder.png</normaloff>icons/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refresh_btn">
       <property name="toolTip">
        <string>Refresh</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/refresh.png</normaloff>icons/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="ChannelParametersWidget" native="true">
       <layout class="QVBoxLayout" name="channelNameLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Measurements</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="TreeWidget" name="measurements_tree">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Available channels:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="channel_A">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="channel_B">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="channel_C">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="channel_D">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="channel_E">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>E</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="channel_F">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>F</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="channel_G">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="channel_H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="PlotWidget" name="graph"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Technical notes</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPlainTextEdit" name="notes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="save_note_btn">
              <property name="text">
               <string>Save note</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
